### PR TITLE
[3D Object Detection] Make iou_3d a function

### PR DESCRIPTION
**Pull request by ianstenbit**

### CODE REVIEW

File: keras_cv/ops/iou_3d.py [hunk 1]
```diff
@@ -17,32 +17,28 @@
...
-        )
-        self.iou_3d = pairwise_iou_op.pairwise_iou3d
-
-    def __call__(self, y_true, y_pred):
-        return self.iou_3d(y_true, y_pred)
+iou_3d = None
+try:
+    keras_cv_custom_ops = load_library.load_op_library(
+        resource_loader.get_path_to_datafile("../custom_ops/_keras_cv_custom_ops.so")
+    )
```
Comment (tanzhenyu) [inline keras_cv/ops/iou_3d.py:24]: @bhack I believe you mentioned something about lazy loading?

Comment (ianstenbit) [reply keras_cv/ops/iou_3d.py]: Thank you both! Switched to lazy loading, using the same LazySO strategy as TFA

**Review by tanzhenyu — APPROVED** [review]
Thank you!

**Pull request merged**